esp/services/ws_fileio/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ include_directories (
          ${HPCC_SOURCE_DIR}/dali/sasha
     )
 
-ADD_DEFINITIONS( -D_USRDLL -DWsFileIO_API_LOCAL )
+ADD_DEFINITIONS( -D_USRDLL -DWsFileIO_API_LOCAL -DESP_CONTROL_WsFileIO)
```

Comment:
@kenrowland Why are some of these "ESP_CONTROL_" when the others are "ESP_SERVICE_"?

Reply:
It should be ESP_SERVICE. Good catch. Probably from having to type it in so many times.